nevergrad/functions/__init__.py

```diff
@@ -5,5 +5,7 @@
 
 from .functionlib import ArtificialFunction as ArtificialFunction
 from .functionlib import FarOptimumFunction as FarOptimumFunction
+from .functionlib import MLTuning as MLTuning
```

Comment:
Since this depends of a non-main dependency, I'd rather it were not imported here by default

Reply:
note: move code into functions/ml/core.py
functions/ml/__init__.py

elsewhere
from function import ml